starlette/templating.py

```diff
@@ -56,6 +56,12 @@ class Jinja2Templates:
     templates = Jinja2Templates("templates")
 
     return templates.TemplateResponse("index.html", {"request": request})
+
+    ..enable_async:
```

Comment:
Starlette is not using Sphinx, you can write a simple docstring and maybe add it to the `docs` directory with `Asynchronous template rendering`.

Reply:
Ok, thanks for your advice. The document has been updated.